src/workbookManager.ts

```diff
@@ -81,6 +97,18 @@ const generateSingleQueryWorkbookFromZip = async (zip: JSZip, query: QueryInfo,
     });
 };
 
+const generateMultipleQueryWorkbookFromZip = async (zip: JSZip, queries: MultipleQueriesInfo, fileConfigs?: FileConfigs, tableData?: TableData): Promise<Blob> => {
+    await xmlPartsUtils.updateWorkbookPowerQueryDocument(zip, queries.loadedQueryName, queries.mashupDocument, queries.connectionOnlyQueryNames);
```

Comment:
Originally is it a query formula? how will it work for multi queries? what is the expected format?